options/locale/locale_en-US.ini

```diff
@@ -2545,7 +2545,7 @@ settings.visibility.private_shortname = Private
 
 settings.update_settings = Update Settings
 settings.update_setting_success = Organization settings have been updated.
-settings.change_orgname_prompt = Note: changing the organization name also changes the organization's URL.
+settings.change_orgname_prompt = Note: Changing the organization name will also change your organization's URL and free the old name.
```

Comment:
why is `Changing` capitalized?

Reply:
Why wouldn't it be? It makes things look more consistent (this is how every other string looks like) and it also doesn't "break the flow".

Reply:
https://www.grammarly.com/blog/capitalization-after-colons/

It is right.